site/cypress/e2e/Cypress-Gradeable/numeric_grading.spec.js

```diff
@@ -0,0 +1,39 @@
+describe('Test that in the sample numeric grading test that', () => {
+  beforeEach(() => {
+    cy.visit(['sample', 'gradeable', 'grading_test', 'grading']);
+    cy.login('ta');
+  });
+
+  it('double click can activate typing mode', () => {
+    cy.get('#cell-1-0-0').dblclick();
+    cy.get('input#cell-1-0-0.option-small-box.cell-grade').should('exist');
```

Comment:
This should be a better selector, consider adding a cypress selector to the html element. https://docs.cypress.io/guides/references/best-practices#Selecting-Elements